notebooks/Model_Performance_Statistics.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
This notebook is specific to coco-cats experiment. Let's wait until we have general solution that allow us to evaluate repeated model training + evaluation runs.